src/pyclass.rs

```diff
@@ -153,6 +153,7 @@ pub trait PyClass:
 struct TypeSlots(Vec<ffi::PyType_Slot>);
 
 impl TypeSlots {
+    #[inline]
```

Comment:
Is this effective?
This is not a public function so the compiler would automatically make this inline if needed.

Reply:
I felt it couldn't hurt to add this attribute here. I can try and measure the benchmarks to see if I can get any meaningful difference.

Reply:
Okay, removed!